arch/riscv/kvm/vcpu_sbi_covg.c

```diff
@@ -95,6 +96,7 @@ static int cove_share_page(struct kvm_vcpu *vcpu, gpa_t gpa,
 	list_add(&tpage->link, &tvmc->shared_pages);
 	spin_unlock(&vcpu->kvm->mmu_lock);
 
+	retdata->out_val = page_to_phys(tpage->page);
```

Comment:
TSM that does not support dynamic page conversion requires information on the physical address of the first shared page. This is communicated as a result from the SBI call.